.gitlab/package_build/deb.yml

```diff
@@ -240,7 +197,7 @@ dogstatsd_deb-x64:
     # Thus, we move the artifacts at the end in a gitlab-friendly dir.
     - *setup_deb_signing_key
     # Use --skip-deps since the deps are installed by `before_script`.
-    - inv -e dogstatsd.omnibus-build --release-version "$RELEASE_VERSION_7" --major-version 7 --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod"
+    - inv -e dogstatsd.omnibus-build --release-version "$RELEASE_VERSION_6" --major-version 6 --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod"
```

Comment:
Same? We did not use to make a difference between dogstatsd agent 6 and agent 7

Reply:
Yes iot and dogstatsd can be dismissed, same as installer, oci